src/io/io.jl

```diff
@@ -38,13 +38,7 @@ function load_restart_file!(dg::AbstractDg, restart_filename)
 
       # Read variable
       println("Reading variables_$v ($name)...")
-      if ndims(dg) == 2
-        dg.elements.u[v, :, :, :] = read(file["variables_$v"])
-      elseif ndims(dg) == 3
-        dg.elements.u[v, :, :, :, :] = read(file["variables_$v"])
-      else
-        error("Unsupported number of spatial dimensions: ", ndims(dg))
-      end
+      dg.elements.u[v, :, ..] = read(file["variables_$v"])
```

Comment:
Sorry, maybe I wasn't clear enough: the `..` should be where the DG nodes are. That is, for `u` we have `[variable_id, i, j, k, element_id]` in 3D, and the ellipsis should cover the nodes i, j, k, i.e., `[variable_id, .., :]`. I think this would be more consistent and also follows some sort of understandable logic.

Reply:
Okay, so always `:, ..]`?